rundeckapp/grails-app/views/user/login.gsp

```diff
@@ -190,6 +191,16 @@
 
       </div>
     </div>
+      <script type="text/javascript">
+          function onLoginClicked() {
+            let lbtn = jQuery("#btn-login")
+            if(jQuery('#login').val() === '') return false
```

Comment:
This will give no feedback if the user press the Log In button without entering a username. I know is a very border case but i think that maybe this case should be handled just like a bad password or no pasword situation (give the "Invalid username and password."message).
I'm not even sure if its necessary to block the submit given the fix below.